app/utils/ImplicitDateFormatter.scala

```diff
@@ -29,18 +29,30 @@ trait ImplicitDateFormatter {
     def toZonedDateTime: ZonedDateTime = ZonedDateTime.parse(s, DateTimeFormatter.ISO_ZONED_DATE_TIME)
   }
 
-  implicit class longDate(d: LocalDate) {
-    def toLongDate: String = d.getDayOfMonth + " " + d.getMonth.getDisplayName(FULL, UK) + " " + d.getYear
-    def toLongDateShort: String = d.getDayOfMonth + " " + d.getMonth.getDisplayName(SHORT, UK) + " " + d.getYear
-    def toLongDateNoYear: String = d.getDayOfMonth + " " + d.getMonth.getDisplayName(FULL, UK)
-  }
-
   implicit class shortDate(date: LocalDate) {
     def toShortDate: String = date.format(DateTimeFormatter.ofPattern("d/MM/uuuu"))
   }
 
-  implicit class longDateTime(dt: LocalDateTime) {
-    def toLongDateTime: String = dt.getDayOfMonth + " " + dt.getMonth.getDisplayName(FULL, UK) + " " + dt.getYear
+  implicit class longDate(d: LocalDate)(implicit messages: Messages) {
+
+    def toLongDateNoYear: String = {
+      val dt = Dates.formatDate(org.joda.time.LocalDate.parse(d.toString))(messages)
+      dt.split(" ")(0) + " " + dt.split(" ")(1);
```

Comment:
Can you remove the semicolon at the end of this line

Reply:
done